Makefile

```diff
@@ -679,16 +679,28 @@ ifeq ($(AUTODESPAWN), true)
 	$(MAKE) -k stop-integration-mongodb_metrics
 endif
 
+.PHONY: start-integration-prometheus stop-integration-prometheus test-integration-prometheus
+start-integration-prometheus:
+	$(CONTAINER_TOOL) run -d --name vector_prometheus --net=host \
```

Comment:
I assume we need `--net=host` here for prometheus to scrape the integration test process.

Reply:
Correct. I hate it, but couldn't figure out another viable way of having prometheus connect to the host, particularly with the constraint that the host:port to connect to has to be embedded in a config file that is used before the integration test runs.

Reply:
You could bind the external Prometheus exporter to the IP of the external `docker0` IP?